jest.text.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   runner: 'jest-runner-executor',
   displayName: 'vale',
   moduleFileExtensions: ['md', 'mdx'],
+  modulePathIgnorePatterns: ['build', 'dist', 'public'],
```

Comment:
It's related to the new package because the transpiled files are put into the `build` folder.

We do the same for other packages that run code in a node environment. I think it's helpful to differentiate it from the `dist` package, where we bundle the code into one file.